doc/source/user_guide/index.rst

```diff
@@ -48,20 +49,27 @@ You can launch Fluent in meshing mode with:
 Setting the ``meshing_mode`` argument to ``False`` launches Fluent in solution mode. 
 
 For more information, see :ref:`ref_user_guide_launch` 
-and the API topic :ref:`ref_launcher_launcher`.
+and :ref:`ref_launcher_launcher`.
 
-PyFluent can create and initialize multiple, independent session objects. Each session
-object provides full access to the Fluent capabilities relevant to the session's current
-mode (solution or meshing).
+You can use PyFluent to create and initialize multiple, independent session objects.
+Each session object provides full access to the Fluent components relevant to the
+session's current mode (solution or meshing).
 
 Solution mode session
 ---------------------
-A solution mode session has an active ``solver`` object, which provides two
-distinct interfaces to the solver. The ``solver`` object has a ``tui`` object,
-which is a complete Python exposure of the Fluent solver's TUI (text 
-user interface). This interface allows straightforward execution of commands and 
-modification of settings in a manner that is familiar to existing Fluent 
-users:
+A solution mode session has an active ``solver`` object that provides two distinct
+interfaces to the solver:
+
+- ``tui`` object
+- ``root`` object
```

Comment:
I should declare that due to recent changes (related to the new launch `mode` argument) this `tui` object is going away and depending on how you launch, it might not be there now.